types/src/messages/client_ds_out.rs

```diff
@@ -151,9 +150,7 @@ pub struct DeleteGroupParamsOut {
 #[derive(Debug, TlsSerialize, TlsSize)]
 #[repr(u8)]
 pub enum DsRequestParamsOut {
-    AddUsers(AddUsersParamsOut),
```

Comment:
Is it ok to remove this variant? It will break the protocol right?

Reply:
It's certainly not backwards compatible if that's what you mean. It shouldn't break the protocol, though. With this PR, user are added using the more general GroupOperation request.

Reply:
Thanks. I meant it being not backwards compatible because the index inside `DsRequestParamsOut` is shifted.